contrib/node/src/python/pants/contrib/node/tasks/node_task.py

```diff
@@ -17,13 +19,17 @@
 from pants.contrib.node.targets.node_test import NodeTest
 
 
+logger = logging.getLogger(__name__)
+
+
 class NodeTask(Task):
 
   @classmethod
   def subsystem_dependencies(cls):
     return super(NodeTask, cls).subsystem_dependencies() + (NodeDistribution.Factory,)
 
-  @memoized_property
+  @property
+  @memoized_method
```

Comment:
Pants has its own built in linting, which doesn't trigger for that issue, as far as I can tell.